py/surveyqa/core.py

```diff
@@ -99,6 +152,9 @@ def change_range(i):
     elif show_summary!="no":
         raise ValueError('show_summary should be "all", "subset", or "no". The value of show_summary was: {}'.format(show_summary))
 
+    nights_sub = sorted(set(exposures_sub['NIGHT']))
+    generate_js(outdir, nights_sub, nights != None)
```

Comment:
Please use a more descriptive function name than `generate_js`.  Generate js for what?